### PR TITLE
Added support for multilanguage boolean parsing

### PR DESCRIPTION
**Pull request by babariviere**
This can be a fix to #3 . Give me your opinion about my code.

### CODE REVIEW

File: src/bool.rs [hunk 1]
```diff
@@ -1,9 +1,59 @@
...
+    English,
+    French,
+    Spanish,
+    Italian,
+}
+
+pub fn enable_lang(lang: Lang) {
+    let mut lock = ENABLED_LANG.lock().unwrap();
+    // Check if lang wasn't already added
+    for l in lock.iter() {
```
Comment (LeopoldArkham) [inline src/bool.rs]: More idiomatically written as ` if lock.iter().any(|e| e == lang) {...}` ?

File: src/bool.rs [hunk 2]
```diff
@@ -1,9 +1,59 @@
 use super::humanize::Humanize;
+use std::sync::Mutex;
 
 static TO_TRUE: [&'static str; 9] = ["1", "yes", "y", "ok", "okay", "continue", "go", "on", "true"];
-
 static TO_FALSE: [&'static str; 7] = ["0", "no", "n", "stop", "end", "off", "false"];
 
+static TO_TRUE_FR: [&'static str; 6] = ["oui", "o", "d'accord", "ok", "vrai", "vraie"];
```
Comment (LeopoldArkham) [inline src/bool.rs]: 1 and 0 should work in every language imo

**Comment by LeopoldArkham** [comment]
Looks good.
How about to_true/false_list also modifies static variables, to avoid generating the lists each time?

Thoughts:
- Should English be enabled by default for simplicity's sake? If so, should it be removable too?
- Certain values (0, 1, and ok at least) should probably work in every language. Make them part of the base lists mentioned above by default?
- Added one nitpick in the diff

I'll be working on the rest of the lib tomorrow and this w-e, we should have this in a publishable state soon.

Oh, don't forget to add yourself as a contributor!

File: src/bool.rs (hunk 1 above)
Comment (babariviere) [reply src/bool.rs]: Yep this id better

**Comment by babariviere** [comment]
I will fix all of that, thanks!

**Comment by babariviere** [comment]
I have updated it, can you review it please?

**Comment by LeopoldArkham** [comment]
Looks good, merged!

Feel free to document public functions and/or update `examples/bool.rs`, but I don't mind doing that myself otherwise, so merged.

Review-wise, I just changed the `_` case of the match in the `gen_list` function to `unimplemented!();`, for correctness' sake, it's better than letting it fail silently.

Thanks for your work!

**Comment by babariviere** [comment]
Okay that make sense 😄 
No problem, I think this was the kind of project that I was looking for

**Comment by LeopoldArkham** [comment]
It is pretty great to get more experience. I was actually surprised this didn't exist yet.

**Comment by babariviere** [comment]
You are right 😄

**Comment by LeopoldArkham** [comment]
Well, I'm gonna press on and try to have this done by monday. Feel free to tackle more, but no obligations 😈

**Comment by babariviere** [comment]
I will 😄 I will have some time this week-end but I will have to work on my exam too 😆

**Comment by LeopoldArkham** [comment]
Yeah that sounds like a decent idea. 😄 Bonne chance.

**Comment by babariviere** [comment]
Thanks 😉

**Pull request merged**